Recursive/KochSnowflake.manual-test.js

```diff
@@ -1,4 +1,6 @@
-import { Vector2, iterate } from './KochSnowflake'
+
+import { iterate } from './KochSnowflake'
+import { Vector2 } from '../../Data-Structures/Vectors/Vector2'
```

Comment:
You only need to go one folder back not 2, so use "../Data-Structures/Vectors/Vector2".

Reply:
The test folder is two folders deep:
./Recursive/test/KochSnowflake.test.js
./Data-Structures/Vectors/Vector2.js